src/Pages/Form/Components/UnderLyingDiseaseSection.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  makeStyles,
+  FormGroup,
+  FormControlLabel,
+  Checkbox,
+  CheckboxProps,
+  withStyles,
+  TextField,
+} from "@material-ui/core";
+import React, { ChangeEvent, useState } from "react";
+
+const useStyles = makeStyles(() => ({
+  checkbox: {
+    color: "#3A3A3D",
+    marginLeft: "32px",
+    marginTop: "48px",
+    fontSize: "20px",
+    lineHight: "26px",
+    letterSpacing: "-0.165px",
+  },
+}));
+
+const PinkCheckbox = withStyles({
+  root: {
+    color: "#DFE1E6",
+    "&$checked": {
+      color: "#EF5DA8",
+    },
+  },
+  checked: {},
+})((props: CheckboxProps) => <Checkbox color="default" {...props} />);
+
+const UnderLyingDiseaseSection: React.FC = () => {
+  const classes = useStyles();
+  const [state, setState] = useState({});
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setState({ ...state, [event.target.name]: event.target.checked });
+  };
+
+  return (
+    <>
+      <FormGroup className={classes.checkbox}>
+        <FormControlLabel
```

Comment:
FormControlLabel should be grouped to Array and use map function to render them. Because they are similar.